docs/ADRs/0006-agile-tooling.md

```diff
@@ -0,0 +1,27 @@
+---
+    title: 0006 Agile tooling
+    adr:
+        author: Berry den Hartog
+        created: 28-Feb-2024
+        status:  accepted
+---
+
+## Context
+
+Our development team wants to enhance transparency and productivity in our software development processes.
+We are using GitHub for version control and collaboration. However, to further streamline our process, there is a need
+to incorporate tooling for managing the effort of our team.
+
+## Decision
+
+We will use GitHub Projects as our agile process tool
+
+## Consequences
+
+GitHub Projects seamlessly integrates with our existing GitHub repositories, allowing us to manage our Agile processes
+within the same ecosystem where our code resides. This integration eliminates the need for additional third-party tools,
+simplifying our workflow.
+
+While GitHub Projects offers valuable features for our Agile processes, it's important to acknowledge that there exists
+a steep learning curve associated with its adoption. Additionally, despite its versatility, there are certain
+functionalities that are still evolving and may not fully meet all of our desired requirements at this time
```

Comment:
```suggestion
functionalities that are still evolving and may not fully meet all of our desired requirements at this time.

To currently  overcome these issues, we use Mattermost Boards with a bare minimum board with two kanban lanes for the topics to discuss and rotation of meeting facilitator.
```

Reply:
ignored, because i remove the last paragraph